ui/imports/shared/controls/AmountInput.qml

```diff
@@ -93,14 +95,14 @@ Input {
             const amount = SQUtils.AmountsArithmetic.fromNumber(
                              amountNumber, d.multiplierIndex)
 
-            if (root.validateMaximumAmount) {
+            if (root.validateMaximumAmount && root.maximumAmount && root.maximumAmount.length > 0) {
```

Comment:
nit: The `&& root.maximumAmount` seems implicit to me what we are trying to achieve. Did you mean `Array.isArray(root.maximumAmount)` ? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray

Reply:
`maximumAmount` is a string, so this is correct imo